packages/gestalt/src/Tag.js

```diff
@@ -52,6 +55,44 @@ type Props = {
   type?: 'default' | 'error' | 'warning',
 };
 
+const applyDensityTheme = (s: 'sm' | 'md' | 'lg') => {
```

Comment:
nit: no need to be cryptic, `size` would be a more expressive variable name here 🙂